docs/Chipyard-Basics/Debugging-RTL.rst

```diff
@@ -0,0 +1,85 @@
+Debugging RTL
+======================
+
+While the packaged Chipyard configs and RTL have been tested to work,
+users will typically want to build custom chips by adding their own
+IP, or by modifying existing Chisel generators. Such changes might introduce
+bugs. This section aims to run through a typical debugging flow
+using Chipyard. We assume the user has a custom SoC configuration,
+and is trying to verify functionality by running some software test.
+
+Waveforms
+---------------------------
+
+The default SW simulators do not dump waveforms during execution. To build
```

Comment:
"software RTL simulators"
(let's use consistent terminology across the docs)

Reply:
Done